sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/options/IndexDocumentsOptions.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.options;
+
+import com.azure.core.annotation.Fluent;
+
+/**
+ * Options for document index operations.
+ */
+@Fluent
+public class IndexDocumentsOptions {
+    private boolean throwOnAnyError = false;
```

Comment:
Should the default be `true`?

Reply:
.NET is using false by default. https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/search/Azure.Search.Documents/src/Options/IndexDocumentsOptions.cs#L19

@tg-msft @heaths could explain the reasoning better.

Reply:
Personally prefer to take `true`, as we have retry helper inside of IndexBatchException.